src/connect.tsx

```diff
@@ -1,22 +1,51 @@
 import * as React from 'react';
 import hoistNonReactStatics from 'hoist-non-react-statics';
-import createContext from 'create-react-context';
+import createContext, {
+  ProviderProps,
+  ConsumerProps,
+} from 'create-react-context';
 import { FormikContext } from './types';
 
-export const {
-  Provider: FormikProvider,
-  Consumer: FormikConsumer,
-} = createContext<FormikContext<any>>({} as any);
+export const { Provider, Consumer } = createContext<FormikContext<any> | void>(
+  undefined
+);
+
+export function FormikProvider<Values>(
+  props: ProviderProps<FormikContext<Values>>
+) {
+  return <Provider value={props.value} children={props.children} />;
```

Comment:
This isn't `{...props}` due to a subtle difference in the typings with `create-react-context`. [PR open](https://github.com/jamiebuilds/create-react-context/pull/21/files) to fix.